resources/repos.json

```diff
@@ -670,6 +670,28 @@
 			}
 		}
 	},
+	"ClayCoffee/ClayTech:master": {
```

Comment:
Can you please move this to the bottom of the file? I'd like to keep the order universal and just append new ones at the end.